InvestopediaApi/ita.py

```diff
@@ -1,261 +1,317 @@
+
+        # Case: No pending trades
+        if soup.find("table", class_="table1") is None:
+            return []
+
+        open_trades_table = soup.find("table", class_="table1").find("tbody")
+        open_trades_list = open_trades_table.find_all("tr", class_="table_data")
+
+        open_trades_raw = []
+        for open_trade in open_trades_list:
+            trade_info_list = open_trade.find_all("td")
+            open_trades_raw.append([i.getText() for i in trade_info_list][2:6])
+
+        open_trades = []
+        for raw_data in open_trades_raw:
+            trade_obj = Trade(
+                date_time=raw_data[0],
+                description=raw_data[1],
+                symbol=raw_data[2],
+                quantity=int(raw_data[3])
+            )
+            open_trades.append(trade_obj)
+
+        return open_trades
+
+    def trade(self, symbol, orderType, quantity, priceType="Market", price=False, duration=Duration.good_cancel):
+        """
+        Executes trades on the platform. See the readme.md file
+        for examples on use and inputs. Returns True if the
+        trade was successful. Else an exception will be
+        raised.
+
+        client.trade("GOOG", Action.buy, 10)
+        client.trade("GOOG", Action.buy, 10, "Limit", 500)
+        """
+
+        br = self.br
+        trade_page = self.fetch('/simulator/trade/tradestock.aspx')
+        trade_form = trade_page.soup.select("form#orderForm")[0]
+
+        # input symbol, quantity, etc.
+        trade_form.select("input#symbolTextbox")[0]["value"] = symbol
+        trade_form.select("input#quantityTextbox")[0]["value"] = str(quantity)
+
+        # input transaction type
+        [option.attrs.pop("selected", "") for option in trade_form.select("select#transactionTypeDropDown")[0]("option")]
+        trade_form.select("select#transactionTypeDropDown")[0].find("option", {"value": str(orderType.value)})["selected"] = True
+
+        # input price type
+        [radio.attrs.pop("checked", "") for radio in trade_form("input", {"name": "Price"})]
+        trade_form.find("input", {"name": "Price", "value": priceType})["checked"] = True
+
+        # input duration type
+        [option.attrs.pop("selected", "") for option in trade_form.select("select#durationTypeDropDown")[0]("option")]
+        trade_form.select("select#durationTypeDropDown")[0].find("option", {"value": str(duration.value)})["selected"] = True
+
+        # if a limit or stop order is made, we have to specify the price
+        if price and priceType == "Limit":
+            trade_form.select("input#limitPriceTextBox")[0]["value"] = str(price)
+
+        elif price and priceType == "Stop":
+            trade_form.select("input#stopPriceTextBox")[0]["value"] = str(price)
+
+        prev_page = br.submit(trade_form, trade_page.url)
+        prev_form = prev_page.soup.find("form", {"name": "simTradePreview"})
+        #br.submit(prev_form, prev_page.url)

```

Comment:
There's an # that shouldn't be in line240 from when I was testing and comparing the two functions.